test/unit/regression/SNYK-JS-AXIOS-1038255.js

```diff
@@ -0,0 +1,61 @@
+// https://snyk.io/vuln/SNYK-JS-AXIOS-1038255
+// https://github.com/axios/axios/issues/3407
+// https://github.com/axios/axios/issues/3369
```

Comment:
It is not a unit test or a regression. I didn't find similar proxy related test cases. Maybe we can name the file as `test/specs/proxy.spec.js` and adding more cases later?

Reply:
I placed the file in the `unit` folder so that is was closer to the other code tested with the node adapter. This test must run with the mocha tests; not the karma/jasmine tests.